File: 02.conditions/07-solution1.cpp
#include <iostream>

using namespace std;

int main(){
    int R, G, Y;
    R = 0;
    G =0;
    Y = 0;

    string sticker;
    cin >> sticker;

    char parameter1, parameter2, parameter3, parameter4, parameter5;
    parameter1 = sticker[0];
    parameter2 = sticker[1];
    parameter3 = sticker[2];
    parameter4 = sticker[3];
    parameter5 = sticker[4];

    if(parameter1 == 'Y'){
        Y += 1;
    }
    if(parameter1 == 'G'){
        G += 1;
    }
    if(parameter1 == 'R'){
        R += 1;
    }

    if(parameter2 == 'Y'){
        Y += 1;
    }
    if(parameter2 == 'G'){
        G += 1;
    }
    if(parameter2 == 'R'){
        R += 1;
    }

    if(parameter3 == 'Y'){
        Y += 1;
    }
    if(parameter3 == 'G'){
        G += 1;
    }
    if(parameter3 == 'R'){
        R += 1;
    }

    if(parameter4 == 'Y'){
        Y += 1;
    }
    if(parameter4 == 'G'){
        G += 1;
    }
    if(parameter4 == 'R'){
        R += 1;
    }

    if(parameter5 == 'Y'){
        Y += 1;
    }
    if(parameter5 == 'G'){
        G += 1;
    }
    if(parameter5 == 'R'){
        R += 1;
    }

//    cout << R << G << Y;

    if((R >= 3) || (R >= 2 && Y >= 2) || (G == 0)){
        cout << "nakhor lite";
    }
    else{
        cout << "rahat baash";
    }
    return 0;
}

File: 02.conditions/python/07_solution1.py
R, G, Y = 0, 0, 0

label = input()

# Count the occurrences of 'R', 'G', and 'Y' in the label string
if label[0] == 'Y':
    Y += 1
elif label[0] == 'G':
    G += 1
elif label[0] == 'R':
    R += 1

if label[1] == 'Y':
    Y += 1
elif label[1] == 'G':
    G += 1
elif label[1] == 'R':
    R += 1

if label[2] == 'Y':
    Y += 1
elif label[2] == 'G':
    G += 1
elif label[2] == 'R':
    R += 1

if label[3] == 'Y':
    Y += 1
elif label[3] == 'G':
    G += 1
elif label[3] == 'R':
    R += 1

if label[4] == 'Y':
    Y += 1
elif label[4] == 'G':
    G += 1
elif label[4] == 'R':
    R += 1


# Determine the output based on the counts
if (R >= 3) or (R >= 2 and Y >= 2) or (G == 0):
    print("nakhor lite")
else:
    print("rahat baash")
